RubeesEat/wwwroot/css/site.css

```diff
@@ -113,6 +110,193 @@ nav > ul > li:hover {
+h6 { font-size: 0.5rem; line-height: 1.6;  letter-spacing: 0; }
+
+/* Forms
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+input[type="number"],
+input[type="text"],
+select {
+    height: 38px;
+    padding: 6px 10px; /* The 6px vertically centers text on FF, ignored by Webkit */
+    background-color: white;
+    border-radius: 4px;
+    max-width: 200px;
+}
+
+input[type="number"]:focus,
+input[type="text"]:focus,
+select:focus {
+    border: 1px solid var(--primary-color);
+    outline: 0;
+}
+
+label {
+    display: block;
+    margin-bottom: .5rem;
+    font-weight: 600;
+}
+
+input[type="checkbox"],
+input[type="radio"] {
+    display: inline;
+}
+
+label > .label-body {
+    display: inline-block;
+    margin-left: .5rem;
+    font-weight: normal;
+}
+
+/* Buttons */
+.button,
+button,
+input[type="submit"],
+input[type="reset"],
+input[type="button"] {
+    display: inline-block;
+    height: 38px;
+    padding: 0 30px;
+    color: var(--primary-color-darker);
+    text-align: center;
+    font-size: 11px;
+    font-weight: 600;
+    line-height: 38px;
+    letter-spacing: .1rem;
+    text-transform: uppercase;
+    text-decoration: none;
+    white-space: nowrap;
+    background-color: transparent;
+    border-radius: 4px;
+    border: 1px solid var(--primary-color-darker);
+    cursor: pointer;
+}
+
+.button:hover,
+button:hover,
+input[type="submit"]:hover,
+input[type="reset"]:hover,
+input[type="button"]:hover
+{
+    color: var(--primary-color);
+    border-color: var(--primary-color);
+    outline: 0; 
+}
+
+.button:focus,
+button:focus,
+input[type="submit"]:focus,
+input[type="reset"]:focus,
+input[type="button"]:focus
+{
+    color: var(--primary-color-lighter);
+    border-color: var(--primary-color-lighter);
+    outline: 0; 
+}
+
+.button:disabled,
+button:disabled,
+input[type="submit"]:disabled,
+input[type="reset"]:disabled,
+input[type="button"]:disabled
+{
+    color:gray;
+    border-color: gray;
+    outline: 0;
+}
+
+.button.button-primary,
+button.button-primary,
+input[type="submit"].button-primary,
+input[type="reset"].button-primary,
+input[type="button"].button-primary {
+    color: var(--secondary-color);
+    background-color: var(--primary-color-darker);
+    border-color: var(--secondary-color); 
+}
+
+.button.button-primary:hover,
+button.button-primary:hover,
+input[type="submit"].button-primary:hover,
+input[type="reset"].button-primary:hover,
+input[type="button"].button-primary:hover {
+    color: var(--secondary-color);
+    background-color: var(--primary-color);
+    border-color: var(--secondary-color); 
+}
+
+a.button.button.button-primary:focus,
+button.button-primary:focus,
+input[type="submit"].button-primary:focus,
+input[type="reset"].button-primary:focus,
+input[type="button"].button-primary:focus {
+    color: var(--secondary-color);
+    background-color: var(--primary-color-lighter);
+    border-color: var(--secondary-color); 
+}
+
+.entryLines {
```

Comment:
The way the rules are formulated with `entryLines` as name suggests that it is not really reusable but should be moved into CSS specific to the site that is using the class.

If you want to, we can create a reusable "list" component but that should be more general and not have too specific styling for the one use case we have right now. So some of the rules might still need to be moved to a site specific CSS file.

Also remember that we can redesign the HTML of the list as well. We don't need to stay with an `<ul>` element and completely restyle it. For the kind of data we have here, a tabular display would be better in my opinion as it would make the amounts align vertically.

Reply:
We use the entrylines again in "ShowBills", and also in an individual Bill (but without the "positive/negative amount class")
![image](https://github.com/user-attachments/assets/d1ab8711-daac-4a3b-9536-0dd23d14c16c)

Reply:
Regarding the tabular display: In DSEAT-32 (https://github.com/re-motion/students-RubeesEat/pull/22) I used flexbox and margin:auto, flex-grow 1, etc. to make the entrylines display like a table.
I did it there rather than here because it was more of a issue with the master layout

Reply:
It turns out that if we use divs instead of li, the divs underneath do not inherit the background color. I'm leaving it as a li for now.
Now we changed it to use grid layout